pkg/crc/machine/machine.go

```diff
@@ -399,11 +400,6 @@ func (client *client) Start(startConfig StartConfig) (StartResult, error) {
 		log.Warnf("Cannot update kubeconfig: %v", err)
 	}
 
-	// Approve the node certificate.
-	if err := cluster.ApproveNodeCSR(ocConfig); err != nil {
-		return startError(startConfig.Name, "Error approving the node csr", err)
-	}
-
```

Comment:
This is now not required because we are using `192.168.126.11` as kubelet IP but before 4.4 timeframe when we had different IP for kubelet then it was required to approve pending node certificate.

Reply:
I'd add an explanation about this in the commit log. Especially as this seems to be silently removed in ` Wait for kubelet certs to be renewed`

Reply:
Added an extra commit for this!